scripts/DBWS-JP/c100420026.lua

```diff
@@ -0,0 +1,64 @@
+--征服斗魂 三一爆发
+--scripted by JoyJ
+function c100420026.initial_effect(c)
+	--Activate
+	local e1=Effect.CreateEffect(c)
+	e1:SetCategory(CATEGORY_TOHAND+CATEGORY_SPECIAL_SUMMON)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCountLimit(1,100420026+EFFECT_COUNT_CODE_OATH)
+	e1:SetProperty(EFFECT_FLAG_CARD_TARGET)
+	e1:SetCode(EVENT_FREE_CHAIN)
+	e1:SetTarget(c100420026.target)
+	e1:SetOperation(c100420026.activate)
+	c:RegisterEffect(e1)
+end
+function c100420026.spfilter(c,e,tp,attr)
+	return c:IsSetCard(0x297) and c:IsCanBeSpecialSummoned(e,0,tp,false,false)
+		and not c:IsAttribute(attr)
+end
+function c100420026.filter(c,e,tp)
+	return c:IsFaceup() c:IsSetCard(0x297) and c:IsAbleToHand()
```

Comment:
redtext: `'end' expected (to close 'function' at line 19) near 'c'`
```suggestion
	return c:IsFaceup() and c:IsSetCard(0x297) and c:IsAbleToHand()
```